spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -28,11 +33,27 @@
       end.to not_change(user.notifications, :count)
     end
 
-    it "sends only 1 notification at a time" do
-      user = create(:user, :with_identity, identities: ["github"], created_at: 1.week.ago)
-      expect do
-        sidekiq_perform_enqueued_jobs { described_class.call(user.id) }
-      end.to change(user.notifications, :count).by(1)
+    it "sends only 1 notification at a time with the right notification" do
+      user = create(:user, :with_identity, identities: ["github"], created_at: 1.day.ago)
+
+      expect { sidekiq_perform_enqueued_jobs { described_class.call(user.id) } }.to change(user.notifications, :count).by(1)
+      expect(user.notifications.last.notifiable).to eq(welcome_broadcast)
+
+      Timecop.travel(1.day.since)
```

Comment:
`Timecop.travel` can take a block that will reset the clock at the end. The issue with using the non block version is that it doesn't turn off the travel for subsequent tests AFAIK. Probably `Timecop.return` as `after { }` is enough though, without restructuring the tests completly

Reply:
Oof, this would've been one annoying bug to deal with if you didn't point this out.